charts/dex.yaml

```diff
@@ -0,0 +1,56 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/infra: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
```

Comment:
Didn't we say we'll need gRPC for #1990?

Reply:
Do we really need GRPC? I'm afraid of getting into an authn/authz  mess, whilst access to the configuration/configmap is much easier to manage.

Reply:
If it's acceptable to restart Dex for each config change, then yes avoid the whole gRPC stuff can be much simpler.